Doc/howto/regex.rst

```diff
@@ -460,6 +474,15 @@ In actual programs, the most common style is to store the
    else:
        print('No match')
 
+Python 3.8 added assignment expressions that shorten the above pattern
+by a line::
+
+   p = re.compile( ... )
+   if (m := p.match( 'string goes here' )):
```

Comment:
```suggestion
   if m := p.match('string goes here'):
```

Reply:
Spaces removed, but I'd like to keep the parens.